src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDoclet.java

```diff
@@ -30,7 +30,14 @@
 import java.nio.file.Files;
 import java.nio.file.InvalidPathException;
 import java.nio.file.Path;
-import java.util.*;
+import java.util.ArrayList;
```

Comment:
This does not seem necessary for this PR and looks like an artifact of editing in an IDE.